Duplicati/Library/Utility/FileBackedList.cs

```diff
@@ -101,8 +107,8 @@ public void Reset()
             }
         }
 
-        private Library.Utility.TempFile m_file;
-        private Stream m_stream;
+        private Library.Utility.TempFile? m_file;
+        private Stream? m_stream;

```

Comment:
Sam as above, it cannot be null and should be readonly.

Reply:
Same thing here
<img width="370" alt="image" src="https://github.com/duplicati/duplicati/assets/60369677/d4a8cd0b-3830-42c3-a52b-d19bb6932c33">

So I'm not entirely sure how to proceed

Reply:
I see, it is lazy initialized, so we cannot make it readonly :'(